heartbeat/heartbeat.py

```diff
@@ -0,0 +1,16 @@
+"""
+Code to keep the lambda function alive.
+"""
+import os
+
+from notifications_python_client.notifications import NotificationsAPIClient
+
+API_KEY = os.getenv("HEARTBEAT_API_KEY")
+BASE_URL = "https://api.notification.canada.ca"
+EMAIL_ADDRESS = "success@simulator.amazonses.com"
+TEMPLATE_ID = "7daa4f22-633b-4a7b-8b20-dfc5f66882bd"
```

Comment:
We should get the `BASE_URL` and `TEMPLATE_ID` from environment as well, so we can use the same code for staging and prod

Reply:
hrm. Should we have two heartbeats for the canary test? explicitly send to both the k8s and lambda apis. probably, especially if there's warm up issues for the lambda. Maye we want `BASE_URL` to be an array that we loop over.